File: hw/ayuquina/u2/hw18ZooInPython/HW18ZooInPython/ec_edu_espe_zoo_model/Condor.py
from datetime import datetime
from Bird import Bird

class Condor(Bird):
    def __init__(self, mountains, id, description, date_onborn, gender, number_of_bones, cage):
        super().__init__(id, description, date_onborn, gender, number_of_bones, cage)
        self._mountains = mountains

    def __str__(self):
        return f"Condor(mountains={self._mountains}, {super().__str__()})"

    def feed(self, food):
        raise NotImplementedError("Not supported yet.")

    @property
    def mountains(self):
        return self._mountains

    @mountains.setter
    def mountains(self, mountains):
        self._mountains = mountains

File: hw/bonifaz/u2/HW18ZooInPythonJavaScript/zoojs/ec.edu.espe.zoo.model/Condor.js
const { Bird } = require('./Bird.js'); // Importar clase Bird desde otro archivo

class Condor extends Bird {
    constructor(mountains, id, description, dateOnborn, gender, numberOfBones, cage) {
        super(id, description, dateOnborn, gender, numberOfBones, cage);
        this.mountains = mountains;
    }

    toString() {
        return `Condor{mountains=${this.mountains}, ${super.toString()}}`;
    }

    Feed(food) {
        throw new Error("Not supported yet.");
    }

    getMountains() {
        return this.mountains;
    }

    setMountains(mountains) {
        this.mountains = mountains;
    }
}

module.exports = { Condor };
